scripts/dataset_processing/tts/thorsten_neutral/get_data.py

```diff
@@ -0,0 +1,263 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
2023 now!

Reply:
😅. Since you pointed it out, I am always wondering,
1. why do not we add a GitHub action to automatically update the dates;
2. usually the copyright is dated as `initial year` - `current year`. Shall we change that?

Reply:
We've historically only changed it when the file itself is changed (and also when we remember...), may have to ask legal about whether the procedure should be different.

Reply:
That would be great! If the legal team approves, we can directly apply a GitHub action to minimize human efforts.